WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/TransactionDetailsViewModel.cs

```diff
@@ -45,11 +53,26 @@ private void UpdateValues(TransactionSummary transactionSummary)
 		BlockHeight = transactionSummary.Height.Type == HeightType.Chain ? transactionSummary.Height.Value : 0;
 		Confirmations = transactionSummary.GetConfirmations();
 		IsConfirmed = Confirmations > 0;
-		Amount = transactionSummary.Amount.Abs().ToString(fplus: false, trimExcessZero: false);
-		AmountText = transactionSummary.Amount < Money.Zero ? "Outgoing" : "Incoming";
+		Amount = transactionSummary.Amount.Abs();
+		AmountText = GetAmountText(transactionSummary);
 		BlockHash = transactionSummary.BlockHash?.ToString();
 	}
 
+	private string GetAmountText(TransactionSummary transactionSummary)
+	{
+		if (transactionSummary.Amount >= Money.Zero)
+		{
+			return "Incoming";
+		}
+
+		if (transactionSummary.IsSelfSpend)
+		{
+			return "Outgoing (transaction fee)";
```

Comment:
```suggestion
			return "Outgoing (Fee)";
```

Reply:
I think "Mining fee" is better, because at coinjoin "Fees" is already been used.
Would be better to differentiate between them imo

Reply:
WDYT @soosr ?

Reply:
IMO `Fee` here is fine, it indicates transactions fee since we are in the `Transaction Details` dialog,